Packs/FortiSIEM/ReleaseNotes/2_0_37.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### FortiSIEM v2
+
+- The fetch mode 'Fetch with Events' will now work only for FortiSiem version 6.6 and below.
+- The command ***fortisiem-event-search*** will now work only for FortiSiem version 6.6 and below.
```

Comment:
Recommend:
The ***fortisiem-event-search*** command will now work only for FortiSiem version 6.6 and earlier.